src/components/MapLayerZoomable/MapLayerZoomable.js

```diff
@@ -194,11 +194,16 @@ export default {
 
   mounted () {
     const map = this.getMap()
-    // We can immediately initialize if we have the map ready
-    if (map && map.isStyleLoaded()) {
-      this.initalize()
-      this.isInitialized = true
+    const tryInitialize = () => {
+      if (map && map.isStyleLoaded()) {
+        // We can immediately initialize if we have the map ready
+        this.initalize()
+        this.isInitialized = true
+      } else {
+        setTimeout(tryInitialize, 200)
+      }
```

Comment:
This is a workaround, otherwise `isStyleLoaded()` is false when trying to add the zoomable layers.

Reply:
I sometimes see people using a "retry" function when something might or might not be ready or fail.

Reply:
`tryInitialize` is basically a retry function, I could rename it, but then the first time you are not really "retrying". In general, unless you add a library, retrying is just done with timeouts.